src/components/shared/Select.jsx

```diff
@@ -0,0 +1,58 @@
+import styled from "styled-components";
+
+function Select({ selectInfo, onInputChange }) {
+  return (
+    <StyledSelect>
+      <label htmlFor={selectInfo.id}>{selectInfo.label}</label>
+      {selectInfo.selects.map(select => (
+        <select
+          key={select.id}
+          id={select.id}
+          onChange={ev => onInputChange(select.id, ev.target.value)}
```

Comment:
```suggestion
          onChange={onInputChange}
```

- `p3`

이벤트 핸들러는 보통 해당 이벤트가 발생했을때 원하는 action을 취하기 위해서 사용하는데, 저는 개인적으로 사용하는 사람마다 목적이 다를 수 있다고 생각합니다!

헌데, 현재 이벤트 핸들러 함수는 인자를 `select.id`와 `ev.target.value`로 제한 해버림으로써 공통 컴포넌트를 만드는 가장 큰 목적인 `재사용성`을 해쳐 버린 것 같습니다.

혹시 함수 자체를 인자로 받아서 해당 이벤트에 할당하는 것은 어떨까요?